packages/documentation-framework/components/footer/footer.js

```diff
@@ -13,7 +13,7 @@ import logo from '../../layouts/logo.svg';
 
 export const Footer = () => (
   <React.Fragment>
-    <PageSection key="footer-1" className="ws-org-pfsite-l-footer">
+    <PageSection isWidthLimited key="footer-1" className="ws-org-pfsite-l-footer">
```

Comment:
Looks like this is creating some padding action for this block since there are custom padding styles defined here https://github.com/patternfly/patternfly-org/blob/5f9afc258ddb1bf29f3e9cb1610bb566b81ada24/packages/documentation-framework/components/footer/footer.css#L1-L9

Easiest fix if we want to leave it the way it is would probably just be to set the right vars there, since the body element inherits the main-section's padding vars.

So like:

```
--pf-c-page__main-section--PaddingTop: var(--pf-global--spacer--xl);
--pf-c-page__main-section--PaddingRight: var(--pf-global--spacer--2xl);
--pf-c-page__main-section--PaddingBottom: var(--pf-global--spacer--2xl);
--pf-c-page__main-section--PaddingLeft: var(--pf-global--spacer--2xl);
```